app/styles/about.sass

```diff
@@ -1,36 +1,323 @@
+@import "app/styles/bootstrap/variables"
+@import "app/styles/mixins"
+@import "app/styles/style-flat"
+
 #about-view
+  #nav-container
+    min-height: 55px
+    nav
+      background: $gold
+      &.affix
+        z-index: 1
+        position: fixed
+        top: 0
+        width: 100%
+      ul
+        padding-left: 0
+        list-style: none
+        li
+          height: 22pt
+          display: inline-block
+          margin: 10px 18px 0 18px
+          a
+            color: white
+            text-transform: uppercase
+            text-decoration: none
+        li.active
+          border-bottom: 4px solid white
+    
+  #jumbotron
+    background-color: $navy
+    background-image: url("/images/pages/about/codebackground_zoom_compressed.png")
+    background-size: cover
+    background-repeat: no-repeat
+    
+    h1
+      color: white
+      margin-top: 100px
+      
+    h2
+      color: white
+      margin-bottom: 100px
+      
+  h3, h4
+    text-align: center
+      
+  .responsive-side-margins, h3, h4, p
+    max-width: 460px
+    margin-left: auto
+    margin-right: auto
+    @media (min-width: $screen-sm-min)
+      max-width: inherit
+      
```

Comment:
.responsive-side-margins keeps a bunch of the text in the page from going full width when in xsmall, so that a lot of the textual/page content is contained in a visual "column" in the middle.